src/Raven.Server/Documents/BackgroundWorkHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Linq;
+using Raven.Client.ServerWide;
+using Raven.Server.Documents.DataArchival;
+using Raven.Server.Documents.Expiration;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents;
```

Comment:
`Raven/Server/Background` seems to be more appropriate location for this helper.

Reply:
done